transactions/transactor.go

```diff
@@ -51,7 +51,7 @@ func (e *ErrBadNonce) Error() string {
 
 // Transactor is an interface that defines the methods for validating and sending transactions.
 type TransactorIface interface {
-	NextNonce(rpcClient rpc.ClientInterface, chainID uint64, from types.Address) (uint64, error)
+	NextNonce(ctx context.Context, rpcClient rpc.ClientInterface, chainID uint64, from types.Address) (uint64, error)
```

Comment:
most reasonable thing would be to add ctx to all functions making an RPC call right? particularly the Tx sending ones. No need to do it in this PR though

Reply:
There are still many things that we should do there and will leave this for one of those.
Some further steps:
- to finish moving community-related txs to the wallet service and the flow we have
- to remove community-related api endpoints
- to remove stickers-related api endpoints (once mobile switches to new ones)
- to remove ens-related api endpoints (once mobile switches to new ones)
- to clean wallet API, remove old unused endpoints
- to clean `PathProcessor` interface, removing `Send` and `BuildTransaction` functions, renaming `BuildTransactionV2` to `BuildTransaction` and ctx you suggested will be added at this point
- to simplify the `Transactor` type